### ISSUE TITLE
Data Mine Planets

### ISSUE THREAD

**Issue by sygmo** [sygmo/cs373-idb#49]
The current table being used isn't the best and would require changing the UML. Either settle for it, or use table at http://en.wikipedia.org/wiki/List_of_gravitationally_rounded_objects_of_the_Solar_System

**Comment by mknocke** [comment]
The two attributes not in the tables for the individual planet page on Wikipedia are:
rings
atmosphere (it kind of is there but would be harder to parse)

However, those attributes are in different tables on Wikipedia.
Since we already have a lot of attributes, it would be alright to remove them to simplify parsing.

**Comment by mknocke** [comment]
Composition is very hard to parse, probably want to remove that too

**Comment by mknocke** [comment]
4ed883f8d184a3672635725082ea2efe8791b07c
